globus_automate_client/token_management.py

```diff
@@ -1,11 +1,11 @@
 import os
 from typing import Optional
 
-from fair_research_login import ConfigParserTokenStorage, NativeClient
-from fair_research_login.exc import AuthFailure, LocalServerError
 from globus_sdk import AccessTokenAuthorizer, GlobusAPIError
 from globus_sdk.exc import AuthAPIError
 
+from fair_research_login import ConfigParserTokenStorage, NativeClient
+from fair_research_login.exc import AuthFailure, LocalServerError
```

Comment:
This is probably what's preventing me from running `globus-automate session`, weird that our editors just decided to move the import around. There don't seem to be any other diffs in this file so let's make sure we drop all references to the library. I'm not even sure if we need this file with the new module but I defer on that to you.